app/tests.py

```diff
@@ -631,6 +631,7 @@ def testLoginRedirectToNext(self):
         self.assertEqual(r.url, "https://app-portal.org/")
         self.assertTrue(r.wsgi_request.user.is_authenticated)
 
```

Comment:
This change to address a long standing TODO of not having to hardcode the SUCCESS_URL_ALLOWED_HOSTS setting.